s4e-backend/src/main/java/pl/cyfronet/s4e/api/ResponseExtender.java

```diff
@@ -0,0 +1,55 @@
+package pl.cyfronet.s4e.api;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import pl.cyfronet.s4e.controller.response.SearchResponse;
+import pl.cyfronet.s4e.util.TimeHelper;
+
+import java.time.ZoneId;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class ResponseExtender {
+    private final TimeHelper timeHelper;
+    private final ObjectMapper objectMapper;
+
+    public SearchResponse map(SceneSearch scene, ZoneId zoneId) {
+        return SearchResponse.builder()
+                .id(scene.getId())
+                .productId(scene.getProductId())
+                .footprint(scene.getFootprint())
+                .metadataContent(getMetadata(scene))
+                .sceneContent(getArtifacts(scene))
+                .timestamp(timeHelper.getZonedDateTime(scene.getTimestamp(), zoneId))
+                .build();
+    }
+
+    private JsonNode getArtifacts(SceneSearch scene) {
+        if (scene.getSceneContent() != null) {
+            try {
+                return objectMapper.readTree(scene.getSceneContent()).get("artifacts");
+            } catch (JsonProcessingException e) {
+                log.error("Cannot parse scene content for id : " + scene.getId());
+                return null;
+            }
+        }
+        return null;
```

Comment:
Only include artifacts names list. SceneResponse::sceneContent should be renamed to ::artifacts